Sources/SwifterSwift/UIKit/UITextFieldExtensions.swift

```diff
@@ -144,18 +144,44 @@ public extension UITextField {
         leftViewMode = .always
     }
 
+    /// SwifterSwift: Add padding to the right of the textfield rect.
+    ///
+    /// - Parameter padding: amount of padding to apply to the right of the textfield rect.
+    func addPaddingRight(_ padding: CGFloat) {
+        let paddingView = UIView(frame: CGRect(x: 0, y: 0, width: padding, height: frame.height))
+        rightView = paddingView
+        rightViewMode = .always
+    }
+    
```

Comment:
You can set your Xcode **Editing** settings to trim whitespace lines. This will help avoid SwiftLint errors.
```suggestion

```